CHANGELOG.md

```diff
@@ -1,3 +1,136 @@
+# 0.26.0 (2018-08-??)
+
+## Language changes
+
+- **(breaking-change)** Revert do not collapse unions for sibling types. ([#6351], thanks @asterite)
+- **(breaking-change)** Restrict `exp.@var` to types in the same hierarchy. ([#6358], thanks @asterite)
```

Comment:
Due recent revert #6484, this is no longer true 😢

Reply:
That's right, but don't be sad :-).

FYI, We are waiting a couple of more PR and small fixes before a final round on the changelog is done and the release process begin.